RobotContainerBase.java

```diff
@@ -46,9 +46,9 @@ public static void checkSubsystemsHealth() {
         // MustangScheduler.getInstance().setDefaultCommand(s,
         // s.getDefaultMustangCommand());
         // }
-      } else if (s.getHealth(false).equals(HealthState.RED)) {
-        MustangScheduler.getInstance().cancel(s.getDefaultMustangCommand());
-        MustangScheduler.getInstance().unregisterSubsystem(s);
+      // } else if (s.getHealth(false).equals(HealthState.RED)) {
+      //   MustangScheduler.getInstance().cancel(s.getDefaultMustangCommand());
```

Comment:
Why was this change made? I'm not saying it's wrong. I just have no context. Please stop using "yes" and the equivalent as a commit message when committing code. Write a brief description that explains what you did so someone scrolling through the history can find things. And when necessary explain why, though why is generally better in the code. Cases like this where the code will no longer exist (or at least shouldn't - we need to lose this habit of commenting things out and then leaving them in there - ahem see lines 44-48) are the exception as if you're removing the code there's generally no logical place to leave a comment.

Why is this whole if/else not removed if it's in fact wrong?